Competition/src/test/java/competition/subsystems/climber/climbercommands/HoldRobotLevelCommandTest.java

```diff
@@ -83,18 +84,18 @@ protected void assertTryingToTiltForward() {
     }
 
     protected void assertTryingToTiltBackward() {
-        assertTrue(rear.leftMotor.getMotorOutputPercent() < -0.1);
-        assertTrue(rear.rightMotor.getMotorOutputPercent() < -0.1);
+        assertFalse(rear.leftMotor.getMotorOutputPercent() < -0.1);
```

Comment:
This is probably a bad idea. Instead of just inverting the behavior, we should be explicit about what we want. For example, I think in the backward case, we now expect the power to be 0, not negative.